gramps_webapi/dbmanager.py

```diff
@@ -38,11 +38,12 @@
 class WebDbManager:
     """Database manager class based on Gramps CLI."""
 
-    ALLOWED_DB_BACKENDS = ["sqlite"]
+    ALLOWED_DB_BACKENDS = ["sqlite", "postgresql"]
 
-    def __init__(self, name: str) -> None:
+    def __init__(self, name: str, password: str) -> None:
```

Comment:
This makes `password` a compulsory argument, but it's not needed for SQLite, so wouldn't it be more appropriate to have `password: Optional[str] = None`?

Reply:
Btw this also makes all the unit tests fail with "TypeError: __init__() missing 1 required positional argument: 'password'"

Reply:
Done... But git isn't showing this change. Should I have made a fixup commit rather than a force update commit?